flow/renderer/pyglet_renderer.py

```diff
@@ -55,8 +75,37 @@ def __init__(self, network, mode,
                  path=HOME+"/flow_rendering",
                  sight_radius=50,
                  show_radius=False,
-                 pxpm=2):
-        """Instantiate a pyglet renderer class."""
+                 pxpm=2,
+                 alpha=1.0):
+        """Initialize Pyglet Renderer.
+
+        Parameters
+        ----------
+        network : list
+            A list of road network polygons
```

Comment:
can you be more specific? maybe just by specifying `list of ...`

Reply:
Added detailed comments.